rails/client/app/bundles/Kaiju/components/Component/components/Placeholder/Placeholder.jsx

```diff
@@ -28,17 +35,10 @@ class Placeholder extends React.Component {
     }
   }
 
-  handleDrop(event) {
-    const target = event.target.getAttribute('data-kaiju-component-id');
-    const data = event.dataTransfer.getData('text');
-    const properties = data.length > 0 ? JSON.parse(data) : null;
-    window.postMessage({ message: 'kaiju-replace', properties, target }, '*');
-  }
```

Comment:
`error  Expected 'this' to be used by class method 'handleDrop'  class-methods-use-this`